modules/ocf_mail/files/spam/logging/log-mail

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
don't need this in python3

did this get added automatically or something?

Reply:
I added this because python gave a warning about the encoding when I ran it due to the `™` in the docstring of `mime_decode`, but it was doing the right thing anyway, so I agree this could just be removed.

Reply:
weird, how were you running it? i haven't seen any warnings like this in python3